frontend/app/routes/_gcweb-app.personal-information.phone-number.confirm.tsx

```diff
@@ -18,14 +18,30 @@ export async function loader({ request }: LoaderFunctionArgs) {
   const userId = await userService.getUserId();
   const userInfo = await userService.getUserInfo(userId);
   const session = await sessionService.getSession(request.headers.get('Cookie'));
+  if(!session.has('newPhoneNumber')) return redirect('/');  //Redirect if newPhoneNumber is not set in session
```

Comment:
A few massive nitpicks:

- Enable "Format on Save" if you're using VSCode, which will reformat this line (among others):
```suggestion
  if (!session.has('newPhoneNumber')) return redirect('/');  //Redirect if newPhoneNumber is not set in session
```

- I feel this line of code you wrote is self-documenting and doesn't require the comment

Reply:
Fixed the formatting, I'll enable auto linting in VSC too

Reply:
My tip with respect to code comments..

If you think your code needs a comment, then consider writing them as `log.debug()` statements.. it will document the code, and also provide potentially useful information when troubleshooting.

Reply:
Hmm it looks like the file hasn't been formatted. I'll approve this PR since my comments are nit picks. Feel free to resolve this comment yourself.